### ISSUE TITLE
Custom date format causes navbar to not show

### ISSUE THREAD

**Issue by iPlexor** [karstenpedersen/obsidian-daily-note-navbar#8]
I've tested in a separate vault and verified the navbar is working with default date format YYYY-MM-DD

Default format passing:
✅ YYYY-MM-DD

My date format is this: YYYY/MM. MMMM/YYYY.MM.DD

Since this introduces a path and a custom date format, I tested each in isolation and both failed:

Tested formats failing:
❌ YYYY.MM.DD 
❌ YYYY/YYYY-MM-DD 

NOTE: I noticed the plugin requires a copy of the **Daily note date format**, so I've been syncing this in each test.

**Comment by iPlexor** [comment]
I really like this extension, and if I can debug this properly later I will PR a fix. 

It's likely not working for many users as a result of this bug.

A previous issue related to the Navbar not displaying was closed without a real solution. The user deleted their .obsidian directory, effectively reverting to the default format which does indeed work.

**Comment by karstenpedersen** [comment]
Thanks a lot for the issue :)

I didn't think about adding dots in the date name, so I split the filenames on dots to remove the file extension. I have now changed it to not do this. I have tested it and could now use `YYYY/MM. MMMM/YYYY.MM.DD` and set the date format in the daily note bar to `YYYY.MM.DD`. You might have to reload Obsidian after updating the plugin to make it work.

Feel free to create more issues or suggestions, if you have any.
